src/shogun/kernel/GaussianKernel.cpp

```diff
@@ -75,23 +75,23 @@ bool GaussianKernel::init(std::shared_ptr<Features> l, std::shared_ptr<Features>
 void GaussianKernel::set_width(float64_t w)
 {
 	require(w>0, "width ({}) must be positive",w);
-	m_log_width = GaussianKernel::to_log_width(w);
+	m_width = w;
 }
 
 SGMatrix<float64_t> GaussianKernel::get_parameter_gradient(Parameters::const_reference param, index_t index)
 {
 	require(lhs, "Left hand side features must be set!");
 	require(rhs, "Rightt hand side features must be set!");
 
-	if (param.first == "log_width")
+	if (param.first == "width")
```

Comment:
this would need a big red label that the gradient computed is not wrt to width but wrt the old log width

Reply:
do you mean with a comment here?

Reply:
yes for example